firmware/base2015/CMakeLists.txt

```diff
@@ -29,16 +21,16 @@ set_target_properties(base2015_elf PROPERTIES EXCLUDE_FROM_ALL TRUE)
 
 # custom target for creating a .bin file from an elf binary
 add_custom_target(base2015
-    arm-none-eabi-objcopy -O binary base2015_elf base2015.bin
-    COMMAND cp ${CMAKE_CURRENT_BINARY_DIR}/base2015.bin ${PROJECT_SOURCE_DIR}/run/
+    arm-none-eabi-objcopy -O binary base2015_elf rj-base.bin
+    COMMAND cp ${CMAKE_CURRENT_BINARY_DIR}/rj-base.bin ${PROJECT_SOURCE_DIR}/run/
     DEPENDS base2015_elf
     COMMENT "objcopying to make MBED-compatible executable"
 )
-set_directory_properties(PROPERTIES ADDITIONAL_MAKE_CLEAN_FILES ${PROJECT_SOURCE_DIR}/run/base2015.bin)
+set_directory_properties(PROPERTIES ADDITIONAL_MAKE_CLEAN_FILES ${PROJECT_SOURCE_DIR}/run/rj-base.bin)
 
 add_custom_target(base2015-prog
-    arm-none-eabi-objcopy -O binary base2015_elf base2015.bin
-    COMMAND ${PROJECT_SOURCE_DIR}/util/base2015-prog.sh ${PROJECT_SOURCE_DIR}/run/base2015.bin
+    arm-none-eabi-objcopy -O binary base2015_elf rj-base.bin
```

Comment:
The prog target shouldn't be doing objcopying - that should all be handled in the `base2015` target.